### ISSUE TITLE
Context.require with a String message

### ISSUE THREAD

**Issue by ICONationDevTeam** [icon-project/goloop#60]
https://github.com/icon-project/goloop/blob/8079044b9014bb7e9b4ccdcb16b8708f0906722e/javaee/api/src/java/score/Context.java#L302

For convenience purposes, would it be possible to add another `require` method that accepts a String message as a parameter for the `AssertionError` raised ?

Such code would be easier to read
```java
Context.require(Context.getCaller().equals(minter.get()), "Only minter can call this method");
```

than the current code : 
```java
if (!(Context.getCaller().equals(minter.get())) {
    Context.revert("Only minter can call this method");
}
```

The implementation should be straight-forward.

```java
public static void require(boolean condition, String message) { 
        if (!condition) {
            throw new AssertionError(message);
        }
}
```

I created that custom require method in a custom "utils" package in my SCORE project, but I think it should be included in the builtin Context methods considering how useful and common it is.

**Comment by sink772** [comment]
Yes, it would be convenient to provide such a method.  I will add the `require(boolean, String)` API to the core soon.

FYI, please note that you can't see the revert message from the TX result even if we provide the additional API.  In goloop, all the revert messages will be stripped out and replaced with something like `Reverted(0)`.  To see the original revert message, you need to use `debug trace` CLI.
https://github.com/icon-project/goloop/blob/master/doc/goloop_cli.md#goloop-debug-trace

**Comment by sink772** [comment]
Fixed in d1bb16e0e0393d04d767485232914be2de8f4529
